buildSrc/build.gradle

```diff
@@ -9,7 +9,6 @@ plugins {
 sourceCompatibility = 1.8
 
 repositories {
-	jcenter()
```

Comment:
Can you please move the work you did with removing `jcenter()` to a separate commit? And add `Closes gh-10253` in the end of the message, so we can address https://github.com/spring-projects/spring-security/issues/10253 separately.

Reply:
Done 👍